java/pom.xml

```diff
@@ -12,8 +13,7 @@ distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
--->
-<project>
+--><project>
```

Comment:
Some formatting looks off. Can you verify that you've run the formatter `https://github.com/google/google-java-format` and also `mvn clean verify` (see earlier comment on `pom.xml` about ensuring `shared-configuration` is set as the parent in the pom.xml)

Reply:
Done